testing/wabt/APKBUILD

```diff
@@ -0,0 +1,32 @@
+# Contributor: Ian Douglas Scott <ian@iandouglasscott.com>
+# Maintainer: Ian Douglas Scott <ian@iandouglasscott.com>
+pkgname=wabt
+pkgver=1.0.11
+pkgrel=0
+pkgdesc="WebAssembly binary toolkit"
+arch="all"
+url="https://github.com/WebAssembly/wabt"
+license="Apache-2.0"
+makedepends=cmake
+subpackages="$pkgname-doc"
+source="$pkgname-$pkgver.tar.gz::$url/archive/$pkgver.tar.gz"
+
+prepare() {
+	default_prepare
```

Comment:
prepare() is defined as `default_prepare` by default, so this whole definition can be removed.